docs/website/contents/for-developers/ProtocolVersioning.md

```diff
@@ -0,0 +1,155 @@
+# Introduction
+
+The Cardano protocol will change over time.
+The community innovates applications and updates feature priorities, researchers refine Ouroboros, and developers improve the code.
+
+This document describes how Cardano accomodates these changes as a community, as a software engineering project, and as a distributed system.
+
+## The Protocol Version
+
+Each change to the Cardano protocol advances through the following steps before influencing the network's chain.
+
+- Someone (you!) specifies a change that is backwards-incompatible but worthwhile.
+  For example, it could be a [Cardano Improvement Proposal](https://cips.cardano.org/) (CIP).
+- The Technical Steering Committee and Ouroboros researchers address the worth, safety, and future consequences of the proposed solution.
+- Cardano developers prepare a software release that includes the changes but only enables them for chains on which the protocol version has been incremented.
+- The community uses on-chain governance to increment the protocol version, thereby putting the change into effect.
+  (Stake pool operators must not vote to increment the protocol version before updating their software!)
+
+Cardano philosophy mandates that the community has the final say regarding if and when to adopt some protocol change.
+They accordingly control the final step above: developers determine the exact meaning of each protocol version, but only the community increments the protocol version on mainnet.
+
+It is crucial that the community does not increment the protocol version before enough nodes have upgraded their software.
+Each protocol version increment corresponds to a _hard fork_, because the changes are backwards-incompatible: nodes running the old code will not be able to adopt blocks minted by the new code.
+Even if some block content happened to be valid in both the old and the new protocol, each software release explicitly refuses any block that (extends a ledger state that) has a protocol version greater than that release was intended to implement — see the MaxMajorPV constant.
+
+Even software released well after mainnet increments the protocol version must implement the protocol both with and without that increment's changes.
+The Cardano node must correctly process blocks with older protocol versions, eg when catching up to mainnet for the first time or after being offline for a long time.
+So every protocol change must remain conditional within all future releases, instead of only within the particular release that mainnet used for that increment of the protocol version.
```

Comment:
What does it mean for a protocol change to be conditional on a (future) release?